nixos/modules/services/web-apps/invidious.nix

```diff
@@ -94,14 +123,22 @@ let
         # Not needed because peer authentication is enabled
         password = lib.mkIf (cfg.database.host == null) "";
       };
+
+      host_binding = cfg.address;
     } // (lib.optionalAttrs (cfg.domain != null) {
       inherit (cfg) domain;
     });
 
-    assertions = [{
-      assertion = cfg.database.host != null -> cfg.database.passwordFile != null;
-      message = "If database host isn't null, database password needs to be set";
-    }];
+    assertions = [
+      {
+        assertion = cfg.database.host != null -> cfg.database.passwordFile != null;
+        message = "If database host isn't null, database password needs to be set";
+      }
+      {
+        assertion = cfg.serviceScale >= 1;
+        message = "Service can't be scaled below one instance";
+      }
```

Comment:
I think we can avoid this by using a uint type for the option.

Reply:
🤔 maybe just set it to somethign cartoonishly large like 1024 but we are trading one downside for another. Either is fine I guess.

Reply:
I'd argue that scaling down to zero is a feature but either case, this is completely enough and sufficient.

Reply:
Well scaling down to zero is basically the same as setting `enable = false`. Except that we're still (optionally) provisioning a local PostgreSQL database and generating an invalid nginx configuration. I don't know what this would be useful for to be honest.